### ISSUE TITLE
Question: How could you change the startup "cube" to a custom glb file object?

### ISSUE THREAD

**Issue by laultman** [microsoft/MixedReality-HolographicRemoting-Samples#87]
Trying to modify the sample player app (OpenXR) for a custom user 3D glb file object instead of the stock cube. The code is not documented to help me understand the flow when the app launches in HoloLens 2. In fact, none of the sample code is documented enough for me to follow what or why something is being done. 
Any help, documentation, pointers, coaching - would be awesome!

**Comment by maapp** [comment]
Hi @laultman,

I guess you mean the 3D launcher cube in the HoloLens shell?
If so we just rely on the default OS generated one but if you want to set your own you can follow this documentation on how to implement 3D app launchers for UWP apps: https://learn.microsoft.com/en-us/windows/mixed-reality/distribute/implementing-3d-app-launchers

**Comment by laultman** [comment]
Thanks @maapp, I did get it working and I did find the documentation you mentioned. For the benefit of others maybe reading this, the documentation uses words that seem to mean they are suggested guidelines. The fact is the specifications are constraints. The number of polygons is no more than 10 thousand and the references to textures are constraints.
I had a 3D model built by the customer that had 52+ thousand polygons with a fairly complex texture having multiple material image files. I had to decimate the model to less than 10k polygons and remove the texture image files. I had to modify the materials to simple colors and gradients. Once my model met the constraints and I renamed it to the name of the current file in the project it worked.
